plugins/org.eclipse.elk.core.service/src/org/eclipse/elk/core/service/graphvisitors/CompoundGraphElementVisitor.java

```diff
@@ -0,0 +1,66 @@
+/*******************************************************************************
+ * Copyright (c) 2020 Kiel University and others.
+ * 
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ * 
+ * SPDX-License-Identifier: EPL-2.0  *******************************************************************************/
+package org.eclipse.elk.core.service.graphvisitors;
```

Comment:
I think this could well end up in the `util` package.

Reply:
I will move it.